.github/workflows/server-release.yml

```diff
@@ -1,8 +1,9 @@
 name: Server Release
 
 on:
-  release:
-    types: [published]
+  push:
+    branches:
+      - main
```

Comment:
🤔

Reply:
Accidentally pushed this here while debugging this on my forked copy.

Reply:
Keep this comment open, so that we remember to fix it before merging.